src/chord/private/chord/local_node.cpp

```diff
@@ -1,486 +1,495 @@
 #include "chord/local_node.h"
 #include "crypto/sha1.h"
```

Comment:
Unnecessary indentation changes causing huge number of lines change. Please rectify that.